pwa/src/components/sources/sourceForm.tsx

```diff
@@ -326,6 +328,7 @@ export const SourceForm: React.FC<SourceFormProps> = ({ sourceId }) => {
                         />
                       </div>
                     </div>
+                    <br />
```

Comment:
Any and all layout should be done with `css` and not with `<br />` elements. Therefore, this `<br />` shouldn't be here.

Note: a `br` element is used to break a line within text, not to create a layout.